### PR TITLE
Tank Fixes

### PR DESCRIPTION
**Pull request by RlonRyan**
Fixes Tanks.

Just have to fix the manual addition stuff.

### CODE REVIEW

**Comment by RlonRyan** [comment]
Should close #382.

File: src/main/java/com/InfinityRaider/AgriCraft/items/blocks/ItemBlockCustomWood.java [hunk 1]
```diff
@@ -39,27 +39,6 @@ public ItemBlockCustomWood(Block block) {
     }
 
     /**
-     * Places the associated block at a location in the world.
```
Comment (RlonRyan) [inline src/main/java/com/InfinityRaider/AgriCraft/items/blocks/ItemBlockCustomWood.java]: This makes it so there is no weird flash of the default material before the actual material shows up.

File: src/main/java/com/InfinityRaider/AgriCraft/tileentity/TileEntityMultiBlock.java [hunk 2]
```diff
@@ -13,11 +13,7 @@
 
 public abstract class TileEntityMultiBlock extends TileEntityCustomWood implements IDebuggable {
 
-	/**
-	 * Represents if the multiblock is pre-1.4
-	 */
-	private boolean oldVersion = false;
-
+	private boolean reform = false;
```
Comment (RlonRyan) [inline src/main/java/com/InfinityRaider/AgriCraft/tileentity/TileEntityMultiBlock.java]: This is the same thing as the oldVersion boolean, just put to more use.

**Comment by RlonRyan** [comment]
Oh the pauses in the water levels reflects the delay in syncs... Perhaps the discrete level syncing isn't fast enough? I might need to adjust the levels...

**Comment by RlonRyan** [comment]
Syncing every 250mB sounds o.k.? (Not to frequent, not to infrequent?)

**Comment by RlonRyan** [comment]
Might also affect #312.

**Comment by RlonRyan** [comment]
Oops. You might want to check the onBlockActivated() method, and somehow get it to just ignore water applications and not everything... :smile:

**Comment by RlonRyan** [comment]
Hmm... I think I am going to change that mass of variables, getters, and setters at the top of the tileentityMultiBlock file to an immutable object.

**Comment by RlonRyan** [comment]
I think I might just have spent an hour that was due to an NBTTag spelling error...

**Comment by RlonRyan** [comment]
I think this works pretty nicely.

**Comment by RlonRyan** [comment]
I'd say that this is a go on the merge.

**Comment by InfinityRaider** [comment]
I'm gna do some testing, but I'm going to wait with merging this into master until after the next release.

**Comment by RlonRyan** [comment]
Ok.

**Comment by RlonRyan** [comment]
Wait, why are we holding off the update?

**Pull request merged**